Firestore/Example/Tests/Local/FSTLevelDBQueryCacheTests.mm

```diff
@@ -47,6 +60,66 @@ - (void)tearDown {
   self.queryCache = nil;
 }
 
+- (void)testMetadataPersistedAcrossRestarts {
+  [self.persistence shutdown];
+  self.persistence = nil;
+
+  DatabaseId database_id{"p", "d"};
```

Comment:
This whole mess (down to line 77 or so and repeated below) is `[FSTPersistenceTestHelpers levelDBPersistence]`

Reply:
Done.